pkg/server/plugin/nodeattestor/awsiid/iid.go

```diff
@@ -51,6 +51,13 @@ var (
 			},
 		},
 	}
+
+	defaultPartition = "aws"
+	partitions       = []string{
```

Comment:
Are these defined as constants anywhere in the AWS Go SDK that we can reference?

Reply:
Not that I could any.

Reply:
Ok, alternatively could we link to the AWS documentation where these strings are defined in a code comment? In case we need to update this set of supported partitions in the future, it would be good to have a reference of where they're defined.

https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html